camoco/COB.py

```diff
@@ -1499,6 +1503,8 @@ def _calculate_gene_hierarchy(self,method='single'):
         dists = (dists * pcc_std) + pcc_mean
         dists = np.tanh(dists)
         dists = 1 - dists
+        #convert nan to 0's, linkage can only use finite values
```

Comment:
We should investigate this some more. Now that I think about it more, there should not be NaNs in the score. I am not sure why they are there.